pkg/traverse/golang.go

```diff
@@ -0,0 +1,23 @@
+package traverse
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// GoBinDir TODO experimental
+func GoBinDir(tc Context) (dir string, err error) {
+	if path, ok := tc.LookupEnv("GOPATH"); ok {
+		dir = strings.Split(path, string(os.PathListSeparator))[0]
+	}
+
```

Comment:
This is fine but

There is also GOBIN environnement variable that needs to be checked before falling back on GOPATH/bin, then to ~/go/bin

https://github.com/golang/go/blob/5123f38e050c5ee7130d459ea247d998a838b5a1/src/cmd/go/internal/work/build.go#L569

Reply:
ah right, seems i forgot that one - thanks